code/script.js

```diff
@@ -1,51 +1,221 @@
+  scoops.addEventListener("change", () => {
+    userResponse(`${scoops.value} scoop(s) please!`);
+    setTimeout(botResponse(`${scoops.value} scoop(s), got it!`), 1000);
+    
+    // Clear the dropdown list and move to next question
+    scoops.remove();
+    setTimeout(() => questionAboutToppings(), 1000);
+  });
 }
 
-// Set up your eventlisteners here
+//------------QUESTION 4-------------
+// Bot shows options for toppings upon receiving user's answer about number of scoops
+const questionAboutToppings = () => {
+  botResponse("What about toppings? Would you like to add some?");
+
+  // Create buttons for toppings
+  inputWrapper.innerHTML += `
+    <button id="sprinkles" value="Sprinkles">Sprinkles</button>
+    <button id="sauce" value="Chocolate sauce">Chocolate sauce</button>
+    <button id="fruit" value="Fresh fruit">Fresh fruit</button>
+    `;
+  
+  // Store the buttons in variables
+  const sprinklesButton = document.getElementById("sprinkles");
+  const sauceButton = document.getElementById("sauce");
+  const fruitButton = document.getElementById("fruit");
+
+  // Function to clear the buttons and move to the next question
+  const moveToLastQuestion = () => {
+    sprinklesButton.remove();
+    sauceButton.remove();
+    fruitButton.remove();
+    setTimeout(() => questionAboutConfirmation(), 1000);
+  };
+
+  // Function to add event listener to click on buttons so user's choice is displayed and bot shows confirmation message
+  const toppingConfirmation = (button) => {
+    button.addEventListener("click", () => {
+      userResponse(`${button.value} please. That's my favourite!`);
+      setTimeout(botResponse(`${button.value} it is!`, "bot"), 1000);
+      moveToLastQuestion();
+    });
+  }
+
+  // Invoke the functions with buttons
+  toppingConfirmation(sprinklesButton);
+  toppingConfirmation(sauceButton);
+  toppingConfirmation(fruitButton);
+}
+
+//------------QUESTION 5-------------
+// Bot asks for final confirmation of the whole order
+const questionAboutConfirmation = () => {
+  botResponse("Great! Would you like to confirm your order?");
+
+  // Create buttons for toppings
+  inputWrapper.innerHTML += `
+    <button id="yes" value="Yes">Yes</button>
+    <button id="no" value="No">No</button>
+    `;
+  
+  // Store the buttons in variables
+  const yesButton = document.getElementById("yes");
+  const noButton = document.getElementById("no");
+  
+  // Reload the page after an amout of timeout
+  const reloadSite = (timeoutPeriod) => {
+    setTimeout("location.reload(true);", timeoutPeriod);
+  }
+
+  // Add event listener to the buttons so user's choice is displayed and bot shows confirmation message
+  yesButton.addEventListener("click", () => {
+    userResponse(`${yesButton.value} please! Can\'t wait!`);
+    setTimeout(botResponse("Thank you. Your ice cream is being prepared. Have a great day!"), 1000);
```

Comment:
if you stored the value now you can write it on the message for e.g your chocolate ice cream with the sprinkles topping will be prepared soon